CIME/test_scheduler.py

```diff
@@ -924,7 +932,9 @@ def _xml_phase(self, test):
                 elif opt.startswith("A"):
                     # A option is for testing in ASYNC IO mode, only available with nuopc driver and pio2
                     envtest.set_test_parameter("PIO_ASYNC_INTERFACE", "TRUE")
-                    envtest.set_test_parameter("CIME_DRIVER", "nuopc")
```

Comment:
This also looked really problematic. Setting the driver to nuopc even if that doesn't match cime_driver?

Reply:
PIO async mode is only supported in nuopc - but since cesm only supports the nuopc driver now, this is okay to remove.